public_html/wp-content/plugins/wordcamp-forms-to-drafts/wordcamp-forms-to-drafts.php

```diff
@@ -42,7 +42,7 @@ public function print_front_end_styles() {
 		?>
 
 		<style>
-			<?php require_once( __DIR__ . '/front-end.css' ); ?>
+			<?php require_once __DIR__ . '/front-end.css'; ?>
```

Comment:
Ah, I've been particularly mindful of this. Because I've turned on saveOnFix, and noticed there weren't many auto-fixes this time so I thought to include them as well. But would you suggest that even if there are just two or three of these linter fixes, they should be placed in a separate PR to avoid confusion?

Reply:
A few lines of linter changes are fine 👍🏻 

In this case I was thinking that changes related to the forms-to-drafts plugin, like the stubs and `wcfd-key` were disconnected enough that they'd make more sense as a separate PR. After taking a second look, though, I think you were right to include it here 👍🏻